src/celllistimpl.h

```diff
@@ -442,12 +444,20 @@ template <typename TMember, typename TIndex> class DenseContainer : virtual publ
         return indices;
     }
 
-    DenseContainer(std::size_t size) { container.resize(size); }
+    DenseContainer(std::size_t size) {
```

Comment:
The DenseContainer throws SIGKILL if `size` is larger than cca 1 billion, so I added a safecheck.